qiskit/optimization/converters/__init__.py

```diff
@@ -30,27 +39,27 @@
 
    InequalityToEquality
    IntegerToBinary
-   QuadraticProgramToQubo
+   IsingToQuadraticProgram
```

Comment:
For the IsingToQP and QPToIsing - these are completely deprecated and will go away. Although you get the warning if you used the code already in the past I tried to make it more visible in the docs by doing this - from RY var form which was deprecated
```
class RY(VariationalForm):
    r"""DEPRECATED The RY Variational Form.

    The RY trial wave function is layers of :math:`y` rotations with entanglements.
```
Here DEPRECATED was added to the start of the main docstring part, to the single line description, so it shows up in the tables. Can we do this for these too now - since we really want to discourage any new use of these.

Reply:
OK. I will add "DEPRECATED" and remove the classes from documentation.

Reply:
Oh ok, I figured they would be in the documentation since we still support them, but since they had the DEPRECATED in there it would be visible to users and hopefully any new users would avoid these. Though I think only the code says what the replacement is, we do not have anything in the docstring in these files to take readers to the right method - but that could be added too with a link to the replacement method.